client/src/App.jsx

```diff
@@ -8,10 +8,10 @@ import { InMemoryCache } from 'apollo-cache-inmemory';
 import { setContext } from 'apollo-link-context';
 import { onError } from 'apollo-link-error';
 import { createUploadLink } from 'apollo-upload-client';
-
+import { MuiPickersUtilsProvider } from '@material-ui/pickers';
```

Comment:
I belive that this dependency has been moved to home, check for it before merging

Reply:
Already fixed in https://github.com/sjdonado/quevent/commit/b21d0ee6cbaaf221f859d6c365bd37f68f0ad404